libogc/sys_report.c

```diff
@@ -0,0 +1,95 @@
+/*-------------------------------------------------------------
+
+osreport.c -- Bulk-only USB mass storage support
```

Comment:
copy paste? :D
pretty sure this isn't ums :p

Reply:
lol, yes copy paste of license text, sorry.